meson.build

```diff
@@ -35,9 +35,6 @@ conf.set_quoted('LOCALSTATEDIR',
 conf.set_quoted('LIBEXECDIR',
                 join_paths(get_option('prefix'),
                            get_option('libexecdir')))
-conf.set_quoted('DATADIR',
```

Comment:
This is a super weird error? Do you have any idea what causes this? (Just in case we need to reintroduce `DATADIR` in future)

Reply:
it get expanded to
```c
typedef IDataObject *LPDATAOBJECT;
typedef enum tagDATADIR {
    DATADIR_GET = 1,
    DATADIR_SET = 2
} 
 "/usr/x86_64-w64-mingw32/sys-root/mingw/include/objidl.h"
```

because objidl.h has:
```c
10666:typedef enum tagDATADIR {
10667:    DATADIR_GET = 1,
10668:    DATADIR_SET = 2
10669:} DATADIR;
```